aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutbox.java

```diff
@@ -321,6 +322,7 @@ private Completable handleIncomingCreate() {
          * Determine which action to take when the incoming mutation type is {@linkplain PendingMutation.Type#UPDATE}.
          * @return A completable with the actions needed to resolve the conflict
          */
+        @SuppressWarnings("unchecked")
```

Comment:
Is this because of the cast to `SerializedModel`?    If so, can you just add a comment indicating that, like:

```suggestion
        @SuppressWarnings("unchecked") // cast to SerializedModel
```

Reply:
cast SerializedModel to Model